.eslintrc

```diff
@@ -33,7 +33,8 @@
   },
   "rules": {
     "func-names": [2, "as-needed"],
-    "no-shadow": 2,
+    "no-shadow": 0,
+    "@typescript-eslint/no-shadow": 2,
```

Comment:
these 2 should be aligned?

Reply:
I don't think so, I was referring to this: https://github.com/typescript-eslint/typescript-eslint/issues/2483#issuecomment-687095358.